spec/capistrano_config_spec.rb

```diff
@@ -0,0 +1,127 @@
+describe Geordi::CapistranoConfig, type: :aruba do
+  subject do
+    Geordi::CapistranoConfig.new('staging', Dir.getwd + '/tmp/aruba')
+  end
+
+  describe '#load_deploy_info' do
+    before { write_file('Capfile', '# Configure your capistrano tasks') }
```

Comment:
Instead of passing a custom root dir to CapistranoConfig, couldn't you set `ENV['RAILS_ROOT']` and have CapistranoConfig find its root itself? This would walk existing code instead of adding a custom workaround for tests only.